regtests/src/bin/main.rs

```diff
@@ -235,6 +235,13 @@ fn test_block<T: GethRPCClient, P: Patch>(client: &mut T, number: usize) {
                                 U256::from_str(&cur_balance).unwrap());
                     }
                 },
+                &AccountChange::Nonexist(address) => {
+                    if !is_miner_or_uncle(client, address, &block) {
```

Comment:
why not if miner or uncle?

Reply:
So this is a special case that only happens when an account is a miner of uncle of that block.

* This is the part of the logic in the regression test that checks account balance matches after EVM finishes.
* Geth only exposes the account state after block/uncle reward finalization, but the account state we built in regression is for account state before it. Everything is the same except balance of block miners and uncles.
* So at that time I decided, this (when an account is a miner or uncle of that block) seldom happens. So we just skip it. Indeed this part can be improved. However, with ECIP1017 it further complicates so it's better to improve this in a better integrated regression model instead of in this JSONRPC regression approach.